lightning-background-processor/src/lib.rs

```diff
@@ -138,11 +123,11 @@ impl BackgroundProcessor {
 	///
 	/// [`Persister::persist_manager`] is responsible for writing out the [`ChannelManager`] to disk, and/or
 	/// uploading to one or more backup services. See [`ChannelManager::write`] for writing out a
-	/// [`ChannelManager`]. See [`FilesystemPersister::persist_manager`] for Rust-Lightning's
+	/// [`ChannelManager`]. See [`Persister::persist_manager`] for Rust-Lightning's
```

Comment:
Oops, I think I was sleepwalking, sorry. Seems strange to link to the trait for "Rust-Lightning's provided implementation", maybe lets just link directly to `FilesystemPersister` with no specific method link? Also, let's call it "LDK's provided implementation" now :)

Reply:
yeah, linking to FilesystemPersister is also kind of strange because they won't find a persist_manager or persist_graph implementation there.

Reply:
also i don't think FilesystemPersister is used in BP anymore and so it complains about no item in scope -- should I just import it only to be used in the comment?

Reply:
Ah, right, we should remove it as a dependency (I think it can be a `dev-dependency` now, no?), then. We can also just say "see the lightning-persister crate", no? I don't think its an issue that we don't link to a specific function, given there's an implementation for the trait mentioned in the previous sentence.